runtime/src/lib.rs

```diff
@@ -767,6 +767,53 @@ impl substrate_pallet_multi_account::Trait for Runtime {
     type MaxSignatories = MultiAccountMaxSignatories;
 }
 
+parameter_types! {
+	pub const SpendPeriod: BlockNumber = 6 * DAYS;
+	pub const Burn: Permill = Permill::from_perthousand(0);
+	pub const TreasuryModuleId: ModuleId = ModuleId(*b"py/trsry");
+
+	pub const TipCountdown: BlockNumber = 1 * DAYS;
```

Comment:
What happens after the day is over? is the tip lost?

Reply:
After a tip receives the threshold of tippers, currently (3/5 council) the tip countdowns starts.
It allows the remaining council members to place a tip as well which would be considered in the average.